developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_30.rst

```diff
@@ -124,5 +125,7 @@ Changed APIs
 Removed APIs
 ^^^^^^^^^^^^
 
+- ``OCP\Util::isValidFileName`` was deprecated in 8.1.0 and is now removed, use either ``OCP\Files\Storage\IStorage::verifyPath`` or the new ``OCP\Files\IFilenameValidator``.
+
```

Comment:
`getForbiddenFileNameChars` was also removed from OCP\Util in the PR?

Reply:
done